catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/view/ModalRegistry.view.js

```diff
@@ -131,6 +125,34 @@ function (ich,Marionette,Backbone,ConfigurationEdit,wreqr,_,$,Utils,Service,moda
                     });
             }
         },
+        urlChanged: function(evt) {
+            var url = evt.target.value;
+            var regExUrl;
+            if (url) {
```

Comment:
If url is not set, hostname won't be set and $registryName.val will be set to the undefined hostname.
This will bypass the checkName which displays an error for empty name.